vcd/sample_vcd_test_config.json

```diff
@@ -57,5 +57,11 @@
     "//": "Size in megabytes",
     "uploadPieceSize": 5,
     "uploadProgress": true
+  },
+  "media": {
+    "mediaPath": "../test-resources/test.iso",
+    "//": "Size in megabytes",
+    "uploadPieceSize": 5,
```

Comment:
Nit: the test.iso is 874 KB, while piece size above is 5 MB. It would be a better test if test.iso was a bit more than 1 MB and piece size 1 MB, so we would test the chunking too.

Reply:
I'll try to search for it, but ISO in "nice" size doesn't easy to find anywhere.

Reply:
Found only 13.3 MB ISO file

Reply:
If you fine with holding 13.3MB file, then I can use it.